tests/x11/ImageMagick.pm

```diff
@@ -26,86 +26,55 @@ use testapi;
 use utils;
 
 
+sub cmpr {
+    my ($og, $cp, $ttl) = @_;
+    my $cmp = script_output("compare -metric rmse $og $cp nullim 2>&1", proceed_on_failure => 1);
+    my ($mse) = $cmp =~ m/(?<=\().*(?=[^(]*\))/g;
```

Comment:
maybe you  could use  -format "%[distortion]\n" and change the output to info, to avoid the regex or check the -format option to see if there's a good way to only show the normalized value (the one inside the parentheses)

Reply:
Thank you for the advice! `%[distortion]` worked like a charm.